### DISCUSSION TITLE
Clickhouse volume size

### DISCUSSION THREAD

**Discussion by switz** [plausible/analytics#2566] [Self-Hosted Support]
My clickhouse volume has grown to 80gb.. that's massive.

Is there any way to prune it? What is is storing specifically? I doubt I need fine grained (just rough) data going back years, so I'd like to get this down to a more manageable size.

**Comment by colinwilson** [comment]
I was just looking at this myself. I think it's mostly due to heavy default log settings.

Here's an good article about it: [Calming down Clickhouse](https://theorangeone.net/posts/calming-down-clickhouse/)

**Reply by ruslandoga** [reply]
And this discussion https://github.com/plausible/analytics/discussions/588 is probably relevant as well. Note however that these configuration options have already been present in the default [docker-compose.yml#L21-L22]( https://github.com/plausible/hosting/blob/a6fe8d13df3566f7bb3625a1a524f5498d56693d/docker-compose.yml#L21-L22) for quite some time. So if anyone is experiencing an excessive disk usage with a recent deployment, it might be due to something else.
